docs/understand/cmake_packages.rst

```diff
@@ -1,139 +1,188 @@
+to make use of CMake's HIP language support will require CMake 3.21 or higher.
+
+Finding Dependencies
+====================
+
+.. note::
+   For a complete
+   reference on how to deal with dependencies in CMake, refer to the CMake docs
+   on `find_package
+   <https://cmake.org/cmake/help/latest/command/find_package.html>`_ and the
+   `Using Dependencies Guide
+   <https://cmake.org/cmake/help/latest/guide/using-dependencies/index.html>`_
+   to get an overview of CMake's related facilities.
+
+In short, CMake supports finding dependencies in two ways:
+
+-  In Module mode, it consults a file ``Find<PackageName>.cmake`` which tries to
+   find the component in typical install locations and layouts. CMake ships a
+   few dozen such scripts, but users and projects may ship them as well.
+-  In Config mode, it locates a file named ``<packagename>-config.cmake`` or
+   ``<PackageName>Config.cmake`` which describes the installed component in all
+   regards needed to consume it.
+
+ROCm dominantly relies on Config mode, one notable exception being the Module
+driving the compilation of HIP programs on Nvidia runtimes. As such, when
+dependencies are not found in standard system locations, one either has to
+instruct CMake to search for package config files in additional folders using
+the ``CMAKE_PREFIX_PATH`` variable (a semi-colon separated list of filesystem
+paths), or using ``<PackageName>_ROOT`` variable on a project-specific basis.
+
+There are nearly a dozen ways to set these variables. One may be more convenient
+over the other depending on your workflow. Conceptually the simplest is adding
+it to your CMake configuration command on the command-line via
+``-D CMAKE_PREFIX_PATH=....`` . AMD packaged ROCm installs can typically be
+added to the config file search paths such as:
+
+-  Windows: ``-D CMAKE_PREFIX_PATH=${env:HIP_PATH}``
+
+-  Linux: ``-D CMAKE_PREFIX_PATH=/opt/rocm``
 
 ROCm provides the respective *config-file* packages, and this enables
 ``find_package`` to be used directly. ROCm does not require any Find module as
 the *config-file* packages are shipped with the upstream projects.
 
-Finding Dependencies
---------------------
+For a complete guide on where and how ROCm may be installed on a system, refer
+to the installation guides in these docs (`Linux <../deploy/linux/index.html>`_).
+
+Using HIP in CMake
+==================
 
-When dependencies are not found in standard locations such as */usr* or
-*/usr/local*, then the ``CMAKE_PREFIX_PATH`` variable can be set to the
-installation prefixes. This can be set to multiple locations with a semicolon
-separating the entries.
+ROCm componenents providing a C/C++ interface support being consumed using any
+C/C++ toolchain that CMake knows how to drive. ROCm also supports CMake's HIP
+language features, allowing users to program using the HIP single-source
+programming model. The HIP API without compiling GPU device code behaves as a
```

Comment:
When a program (or translation-unit) uses the HIP API without compiling any GPU device code, HIP can be treated in CMake as a simple C/C++ library.

Reply:
Suggestion applied.